docs/book/src/clusteropenstack/configuration.md

```diff
@@ -231,6 +231,22 @@ spec:
 
 Any such ports are created in addition to ports used for connections to networks or subnets.
 
+Also, `port security` can be applied to specific port to enable/disable the `port security` on that port; When not set, it takes the value of the corresponding field at the network level.
+
+```yaml
+apiVersion: infrastructure.cluster.x-k8s.io/v1alpha4
+kind: OpenStackMachine
+metadata:
+  name: <cluster-name>-controlplane
+  namespace: <cluster-name>
+spec:
+  ports:
+  - networkId: <your-network-id>
+    ...
+    disablePortSecurity: True
```

Comment:
```suggestion
    disablePortSecurity: true
```

We always write booleans in lower case in yaml. I would recommend keeping it consistent.